Cargo.toml

```diff
@@ -10,9 +10,15 @@ license = "ISC"
 edition = "2018"
 
 [dependencies]
-bare-metal = "0.2.4"
-riscv = "0.6.0"
-vcell = "0.1.2"
+bare-metal = "1.0.0"
```

Comment:
Apparently `bare-metal` is not used anymore.

Reply:
Ok, I can remove it.